features/git-town-new-pull-request/self_hosted.feature

```diff
@@ -0,0 +1,45 @@
+Feature: git-town new-pull-request: when origin is a self hosted servie
+
+  As a developer trying to create a pull request when using a self hosted service
+  I want to be able to configure git town with the code hosting driver it should use
+  So that new-pull-request works with my self hosted service
+
+
+  Background:
+    Given I have "open" installed
+
+
+  Scenario: self hosted bitbucket
+    Given I have a feature branch named "feature"
+    And my remote origin is "git@self-hosted-bitbucket:Originate/git-town.git"
+    And I configure "git-town.code-hosting-driver" as "bitbucket"
+    And I am on the "feature" branch
+    When I run `git-town new-pull-request`
+    Then I see a new pull request with this url in my browser:
+      """
+      https://self-hosted-bitbucket/Originate/git-town/pull-request/new?dest=Originate%2Fgit-town%3A%3Amain&source=Originate%2Fgit-town%.*%3Afeature
+      """
+
+
+  Scenario: self hosted github
+    Given I have a feature branch named "feature"
+    And my remote origin is "git@self-hosted-github:Originate/git-town.git"
+    And I configure "git-town.code-hosting-driver" as "github"
+    And I am on the "feature" branch
+    When I run `git-town new-pull-request`
+    Then I see a new pull request with this url in my browser:
+      """
+      https://self-hosted-github/Originate/git-town/compare/feature?expand=1
+      """
+
+
+  Scenario: self hosted gitlab
```

Comment:
Thoughts about using a Scenario Outline here? I know it would lead to wide tables, but now we have pretty repetitive scenarios.

Alternatively, you could move more steps like `Given I have a feature branch named "feature" ` into the `Background` block.